phpBB/phpbb/template/twig/extension.php

```diff
@@ -163,6 +163,7 @@ public function loop_subset(\Twig\Environment $env, $item, $start, $end = null,
 		// We always include the last element (this was the past design)
 		$end = ($end == -1 || $end === null) ? null : $end + 1;
 
+		/** @psalm-suppress UndefinedFunction */
```

Comment:
Not sure why this is needed?

Reply:
Because psalm throws error about undefined `twig_slice` function.